### ISSUE TITLE
Nebula agent unable to start up after typing in invalid command in the sshd

### ISSUE THREAD

**Issue by ghost** [slackhq/nebula#250]
After accidentally typing in an invalid command in the sshd, Nebula barfed and the process exited. I don't remember the command I ran, but it was either create|print tunnel or query-lighthouse using an invalid IP. 

```
Jul 01 17:54:16  systemd[1]: nebula.service: main process exited, code=exited, status=2/INVALIDARGUMENT
Jul 01 17:54:16  systemd[1]: Unit nebula.service entered failed state.
Jul 01 17:54:16 systemd[1]: nebula.service failed.
```

Cool. Now I'm unable to start Nebula again. Here's what I see:
```
Jun 29 04:16:34 nebula[18321]: runtime: goroutine stack exceeds 1000000000-byte limit
Jun 29 04:16:34 nebula[18321]: fatal error: stack overflow
Jun 29 04:16:34 nebula[18321]: runtime stack:
Jun 29 04:16:34 nebula[18321]: runtime.throw(0xb57b72, 0xe)
Jun 29 04:16:34 nebula[18321]: runtime/panic.go:774 +0x72
Jun 29 04:16:34  nebula[18321]: runtime.newstack()
Jun 29 04:16:34  nebula[18321]: runtime/stack.go:1046 +0x6e9
Jun 29 04:16:34  nebula[18321]: runtime.morestack()
Jun 29 04:16:34  nebula[18321]: runtime/asm_amd64.s:449 +0x8f
Jun 29 04:16:34  nebula[18321]: goroutine 39 [running]:
Jun 29 04:16:34  nebula[18321]: github.com/slackhq/nebula.(*LightHouse).Query(0xc000022300, 0x0, 0xc57c60, 0xc000232000, 0x0, 0x0, 0x0, 0x0, 0x0)
```
[nebula_crash_logs.txt](https://github.com/slackhq/nebula/files/4861460/nebula_crash_logs.txt)

1. Should the process terminate on an invalid command from sshd?
2. How can I get Nebula working again?

EDIT: #2 was addressed in #170, thanks Wade!

**Comment by nbrownus** [comment]
Which version of nebula are you running?

**Comment by brianluong4765** [comment]
1.1.0

**Comment by wadey** [comment]
Please upgrade to 1.2.0, the crash you have was fixed with #170. Although this crash also means that you probably have an invalid Nebula config (you have a lighthouse IP defined that is not part of the Nebula subnet in your cert)

The other bug in sshd we should look into.

**Comment by brianluong4765** [comment]
Ah, that was the issue, thanks Wade. Sorry to link two unrelated issues together! I will update the description.
